plugins/pulp_rpm/plugins/importers/iso/sync.py

```diff
@@ -160,6 +160,7 @@ def download_succeeded(self, report):
                     iso.save()
                 except NotUniqueError:
                     iso = iso.__class__.objects.filter(**iso.unit_key).first()
+                self.add_catalog_entries([iso])
```

Comment:
No, just when downloaded.
Good question.  I can't think of a reason we'd add an entry if we successfully downloaded the file.  I will remove adding the catalog entries here and in the YUM importer download flow.